index.js

```diff
@@ -1 +1,73 @@
-console.log('hello world');
+import { add, edit, read, remove, editOneElement } from "./jsonFileStorage.js";
+
+/**
+ * moves an item from one array of the JSON object to the end of another array
+ * @param {string} filename
+ * @param {string} sourceKey - The name of the key of the array we wish to take the task from
+ * @param {number} sourceArrayIndex - The index of the array we wish to edit
+ * @param {string} targetKey - The name of the key of the array we wish to push the task to
+ * @param {function} callback
+ */
+const complete = function (
+  filename,
+  sourceKey,
+  sourceArrayIndex,
+  targetKey,
+  callback
+) {
+  let movedTaskString;
```

Comment:
what does this variable do here?

Reply:
i assign `movedTaskString `to the string of the task to be marked complete (line 24), and later push that to the target array in `jsonContentObj `(line 27). rather than pushing the task string immediately, i chose to assign it to a variable first to make it more readable.

Reply:
nice 👍  perhaps you could consider moving that variable inside the `if` statement if it isn't being used elsewhere